hummingbot/connector/exchange/bittrex/bittrex_auth.py

```diff
@@ -59,8 +52,11 @@ def construct_content_hash(body: Dict[str, any] = {}) -> Tuple[str, bytes]:
             "Content-Type": "application/json",
             "Accept": "application/json",
```

Comment:
These are not authentication headers and should not be included here (they are managed by the RESTAssistant)

Reply:
Removed the headers